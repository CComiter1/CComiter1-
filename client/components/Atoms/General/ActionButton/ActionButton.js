import styled from 'styled-components'

export default styled.button`
  font-family: ${({ theme }) => theme.bodyFontFamily};
  background-color: transparent;
  border: 3px solid ${({ theme }) => theme.accentColor};
  font-size: 1.3rem;
  padding: 1rem 1.5rem;
  border-radius: 100rem;
  color: ${({ theme }) => theme.accentColor};
  &: hover {
    color: white;
    background-color: ${({ theme }) => theme.accentColor};
  }
`
