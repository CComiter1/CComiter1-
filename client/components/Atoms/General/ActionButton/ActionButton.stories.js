import React from 'react'
import { storiesOf } from '@storybook/react'

import ActionButton from './ActionButton'

storiesOf('Atoms/General/ActionButton', module)
  .add('Regular', () => <ActionButton>Click</ActionButton>)
