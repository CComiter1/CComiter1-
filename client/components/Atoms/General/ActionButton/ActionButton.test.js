import React from 'react'
import renderer from 'react-test-renderer'

import ActionButton from './ActionButton'


describe('Action Button', () => {
  const component = renderer.create(<ActionButton>Press Me</ActionButton>)
  expect(component.toJSON()).toMatchSnapshot()
})
