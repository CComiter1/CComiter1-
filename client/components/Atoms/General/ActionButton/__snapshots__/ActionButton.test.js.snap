// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<button
  className="sc-bdVaJa iAMTQB"
>
  Press Me
</button>
`;
